rioxarray/_io.py

```diff
@@ -1263,6 +1263,12 @@ def open_rasterio(
     if chunks is not None:
         result = _prepare_dask(result, riods, filename, chunks)
 
+    result.encoding["preferred_chunks"] = {
```

Comment:
Should this be moved into the `if chunks is not None:` block?

Reply:
On the contrary, I think this could be moved to an `else` section. This is need by xarray when using the backend, so `chunks` will be `None` in that case

Reply:
Makes sense to me 👍